packages/babylon/src/parser/statement.js

```diff
@@ -1310,21 +1310,44 @@ export default class StatementParser extends ExpressionParser {
       this.parseExportStar(node);
       if (node.type === "ExportAllDeclaration") return node;
     } else if (this.isExportDefaultSpecifier()) {
-      this.expectPlugin("exportDefaultFrom");
       const specifier = this.startNode();
       specifier.exported = this.parseIdentifier(true);
+      let isDefaultfrom = false;
+      if (specifier.exported.name === "default" && this.isContextual("from")) {
+        isDefaultfrom = true;
+        const lookaheadState = this.lookahead();
+        if (
+          lookaheadState.value === "=" ||
```

Comment:
What if I have another token? e.g. `export default from + 2;`

Reply:
Yeah Sorry, didn't consider that.. Will update the PR